daml-assistant/integration-tests/BUILD.bazel

```
@@ -89,9 +95,9 @@ da_haskell_test(
         "//compiler/damlc/tests:generate-simple-dalf",
         "@mvn_dev_env//:mvn",
         "@tar_dev_env//:tar",
-    ] + ([] if is_windows else ["//language-support/ts/daml-types:npm_package"]),
+    ] + ([] if is_windows else ts_libraries),
     # I’m sure the mvn stuff will be flaky.
-    flaky = True,
+    # flaky = True,
```

```suggestion
    flaky = True,
```
I’m pretty sure this is still flaky

---

Oops, I wanted my tests to fail faster during development. I'll unremove it.

---

Good catch!